GliaWidgets/Interactor/Interactor.swift

```diff
@@ -264,8 +265,16 @@ extension Interactor: CoreSdkClient.Interactable {
     }
 
     var onEngagementRequest: CoreSdkClient.RequestOfferBlock {
+        print("Engagement request received")
```

Comment:
let's use `debugPrint` until we will have a proper logging mechanism.

Reply:
fixed [682819e](https://github.com/salemove/ios-sdk-widgets/pull/442/commits/682819e8bbf2dda3a7c51b7e9495e5c8761701d2)